New mamba release v1.0.0 broke CI

**Issue**
New mamba release v1.0.0 broke CI. It leads to problem in the dependency tree, ultimately `mamba-0.1.2` is installed. 
Last working version v0.27.0.
_Error from CI_
```
2022-11-02T14:06:14.7048504Z ##[group]Run conda install -y mamba
2022-11-02T14:06:14.7048831Z [36;1mconda install -y mamba[0m
2022-11-02T14:06:14.7107533Z shell: /usr/bin/bash -e {0}
2022-11-02T14:06:14.7107753Z env:
2022-11-02T14:06:14.7107962Z   PS1: (__setup_conda) 
2022-11-02T14:06:14.7108232Z   CONDA_PREFIX: /usr/share/miniconda/envs/__setup_conda
2022-11-02T14:06:14.7108495Z   CONDA_DEFAULT_ENV: __setup_conda
2022-11-02T14:06:14.7108750Z   CONDA_PROMPT_MODIFIER: (__setup_conda) 
2022-11-02T14:06:14.7109019Z   CONDA_EXE: /usr/share/miniconda/bin/conda
2022-11-02T14:06:14.7109236Z   _CE_M: 
2022-11-02T14:06:14.7109420Z   _CE_CONDA: 
2022-11-02T14:06:14.7109665Z   CONDA_PYTHON_EXE: /usr/share/miniconda/bin/python
2022-11-02T14:06:14.7109904Z ##[endgroup]
2022-11-02T14:06:17.4410492Z Collecting package metadata (current_repodata.json): ...working... done
2022-11-02T14:06:19.6886730Z Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
2022-11-02T14:06:22.6296306Z Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.
2022-11-02T14:07:31.9629341Z Collecting package metadata (repodata.json): ...working... done
2022-11-02T14:07:47.4680132Z Solving environment: ...working... done
2022-11-02T14:07:47.5059312Z 
2022-11-02T14:07:47.5059676Z 
2022-11-02T14:07:47.5060430Z ==> WARNING: A newer version of conda exists. <==
2022-11-02T14:07:47.5060891Z   current version: 4.12.0
2022-11-02T14:07:47.5061535Z   latest version: 22.9.0
2022-11-02T14:07:47.5061827Z 
2022-11-02T14:07:47.5062282Z Please update conda by running
2022-11-02T14:07:47.5062582Z 
2022-11-02T14:07:47.5063309Z     $ conda update -n base -c defaults conda
2022-11-02T14:07:47.5063641Z 
2022-11-02T14:07:47.5063985Z 
2022-11-02T14:07:47.5272326Z 
2022-11-02T14:07:47.5272967Z ## Package Plan ##
2022-11-02T14:07:47.5273601Z 
2022-11-02T14:07:47.5273972Z   environment location: /usr/share/miniconda/envs/__setup_conda
2022-11-02T14:07:47.5274590Z 
2022-11-02T14:07:47.5274895Z   added / updated specs:
2022-11-02T14:07:47.5275711Z     - mamba
2022-11-02T14:07:47.5276031Z 
2022-11-02T14:07:47.5276421Z 
2022-11-02T14:07:47.5276739Z The following packages will be downloaded:
2022-11-02T14:07:47.5277324Z 
2022-11-02T14:07:47.5277621Z     package                    |            build
2022-11-02T14:07:47.5278419Z     ---------------------------|-----------------
2022-11-02T14:07:47.5279103Z     brotlipy-0.7.0             |py37h540881e_1004         342 KB  conda-forge
2022-11-02T14:07:47.5280006Z     certifi-2022.9.24          |     pyhd8ed1ab_0         155 KB  conda-forge
2022-11-02T14:07:47.5280665Z     cffi-1.15.1                |   py37h43b0acd_1         227 KB  conda-forge
2022-11-02T14:07:47.5281566Z     charset-normalizer-2.1.1   |     pyhd8ed1ab_0          36 KB  conda-forge
2022-11-02T14:07:47.5282240Z     colorama-0.4.6             |     pyhd8ed1ab_0          25 KB  conda-forge
2022-11-02T14:07:47.5283132Z     conda-4.12.0               |   py37h89c1867_0         1.0 MB  conda-forge
2022-11-02T14:07:47.5283836Z     conda-package-handling-1.9.0|   py37h540881e_0         1.0 MB  conda-forge
2022-11-02T14:07:47.5284778Z     cryptography-38.0.2        |   py37h5994e8b_1         1.5 MB  conda-forge
2022-11-02T14:07:47.5285421Z     idna-3.4                   |     pyhd8ed1ab_0          55 KB  conda-forge
2022-11-02T14:07:47.5286293Z     libsolv-0.7.22             |       h6239696_0         443 KB  conda-forge
2022-11-02T14:07:47.5286937Z     mamba-0.1.2                |   py37h99015e2_0         104 KB  conda-forge
2022-11-02T14:07:47.5287806Z     pycosat-0.6.4              |   py37h540881e_0         107 KB  conda-forge
2022-11-02T14:07:47.5288444Z     pycparser-2.21             |     pyhd8ed1ab_0         100 KB  conda-forge
2022-11-02T14:07:47.5289333Z     pyopenssl-22.1.0           |     pyhd8ed1ab_0         122 KB  conda-forge
2022-11-02T14:07:47.5290307Z     pysocks-1.7.1              |   py37h89c1867_5          28 KB  conda-forge
2022-11-02T14:07:47.5291854Z     python_abi-3.7             |          2_cp37m           4 KB  conda-forge
2022-11-02T14:07:47.5292431Z     requests-2.28.1            |     pyhd8ed1ab_1          53 KB  conda-forge
2022-11-02T14:07:47.5293001Z     ruamel_yaml-0.15.80        |py37h540881e_1007         259 KB  conda-forge
2022-11-02T14:07:47.5293553Z     tqdm-4.64.1                |     pyhd8ed1ab_0          82 KB  conda-forge
2022-11-02T14:07:47.5294221Z     urllib3-1.26.11            |     pyhd8ed1ab_0         102 KB  conda-forge
2022-11-02T14:07:47.5294760Z     yaml-0.2.5                 |       h7f98852_2          87 KB  conda-forge
2022-11-02T14:07:47.5295417Z     ------------------------------------------------------------
2022-11-02T14:07:47.5295857Z                                            Total:         5.8 MB
2022-11-02T14:07:47.5296318Z 
2022-11-02T14:07:47.5296590Z The following NEW packages will be INSTALLED:
2022-11-02T14:07:47.5296795Z 
2022-11-02T14:07:47.5297292Z   brotlipy           conda-forge/linux-64::brotlipy-0.7.0-py37h540881e_1004
2022-11-02T14:07:47.5297928Z   certifi            conda-forge/noarch::certifi-2022.9.24-pyhd8ed1ab_0
2022-11-02T14:07:47.5298573Z   cffi               conda-forge/linux-64::cffi-1.15.1-py37h43b0acd_1
2022-11-02T14:07:47.5299270Z   charset-normalizer conda-forge/noarch::charset-normalizer-2.1.1-pyhd8ed1ab_0
2022-11-02T14:07:47.5299953Z   colorama           conda-forge/noarch::colorama-0.4.6-pyhd8ed1ab_0
2022-11-02T14:07:47.5300583Z   conda              conda-forge/linux-64::conda-4.12.0-py37h89c1867_0
2022-11-02T14:07:47.5301334Z   conda-package-han~ conda-forge/linux-64::conda-package-handling-1.9.0-py37h540881e_0
2022-11-02T14:07:47.5302091Z   cryptography       conda-forge/linux-64::cryptography-38.0.2-py37h5994e8b_1
2022-11-02T14:07:47.5302730Z   idna               conda-forge/noarch::idna-3.4-pyhd8ed1ab_0
2022-11-02T14:07:47.5303330Z   libsolv            conda-forge/linux-64::libsolv-0.7.22-h6239696_0
2022-11-02T14:07:47.5303957Z   mamba              conda-forge/linux-64::mamba-0.1.2-py37h99015e2_0
2022-11-02T14:07:47.5304600Z   pycosat            conda-forge/linux-64::pycosat-0.6.4-py37h540881e_0
2022-11-02T14:07:47.5305243Z   pycparser          conda-forge/noarch::pycparser-2.21-pyhd8ed1ab_0
2022-11-02T14:07:47.5305897Z   pyopenssl          conda-forge/noarch::pyopenssl-22.1.0-pyhd8ed1ab_0
2022-11-02T14:07:47.5306535Z   pysocks            conda-forge/linux-64::pysocks-1.7.1-py37h89c1867_5
2022-11-02T14:07:47.5307155Z   python_abi         conda-forge/linux-64::python_abi-3.7-2_cp37m
2022-11-02T14:07:47.5307773Z   requests           conda-forge/noarch::requests-2.28.1-pyhd8ed1ab_1
2022-11-02T14:07:47.5308468Z   ruamel_yaml        conda-forge/linux-64::ruamel_yaml-0.15.80-py37h540881e_1007
2022-11-02T14:07:47.5309075Z   tqdm               conda-forge/noarch::tqdm-4.64.1-pyhd8ed1ab_0
2022-11-02T14:07:47.5309689Z   urllib3            conda-forge/noarch::urllib3-1.26.11-pyhd8ed1ab_0
2022-11-02T14:07:47.5310297Z   yaml               conda-forge/linux-64::yaml-0.2.5-h7f98852_2
2022-11-02T14:07:47.5310590Z 
2022-11-02T14:07:47.5310684Z 
2022-11-02T14:07:47.5310689Z 
2022-11-02T14:07:47.5310828Z Downloading and Extracting Packages
2022-11-02T14:07:47.5311016Z 
2022-11-02T14:07:47.6291823Z cryptography-38.0.2  | 1.5 MB    |            |   0% 
2022-11-02T14:07:47.9028548Z cryptography-38.0.2  | 1.5 MB    | 1          |   1% 
2022-11-02T14:07:47.9032152Z cryptography-38.0.2  | 1.5 MB    | ########## | 100% 
2022-11-02T14:07:47.9034967Z cryptography-38.0.2  | 1.5 MB    | ########## | 100% 
2022-11-02T14:07:47.9037401Z 
2022-11-02T14:07:47.9606534Z pyopenssl-22.1.0     | 122 KB    |            |   0% 
2022-11-02T14:07:47.9607415Z pyopenssl-22.1.0     | 122 KB    | ########## | 100% 
2022-11-02T14:07:47.9607841Z 
2022-11-02T14:07:48.0209334Z idna-3.4             | 55 KB     |            |   0% 
2022-11-02T14:07:48.0209990Z idna-3.4             | 55 KB     | ########## | 100% 
2022-11-02T14:07:48.0210611Z 
2022-11-02T14:07:48.0701695Z requests-2.28.1      | 53 KB     |            |   0% 
2022-11-02T14:07:48.0702807Z requests-2.28.1      | 53 KB     | ########## | 100% 
2022-11-02T14:07:48.0736621Z 
2022-11-02T14:07:48.1208341Z mamba-0.1.2          | 104 KB    |            |   0% 
2022-11-02T14:07:48.1211491Z mamba-0.1.2          | 104 KB    | ########## | 100% 
2022-11-02T14:07:48.1211983Z 
2022-11-02T14:07:48.2120264Z brotlipy-0.7.0       | 342 KB    |            |   0% 
2022-11-02T14:07:48.2121363Z brotlipy-0.7.0       | 342 KB    | ########## | 100% 
2022-11-02T14:07:48.2121877Z 
2022-11-02T14:07:48.2628501Z pysocks-1.7.1        | 28 KB     |            |   0% 
2022-11-02T14:07:48.2630067Z pysocks-1.7.1        | 28 KB     | ########## | 100% 
2022-11-02T14:07:48.2630538Z 
2022-11-02T14:07:48.3134317Z pycosat-0.6.4        | 107 KB    |            |   0% 
2022-11-02T14:07:48.3137260Z pycosat-0.6.4        | 107 KB    | ########## | 100% 
2022-11-02T14:07:48.3137454Z 
2022-11-02T14:07:48.3673971Z colorama-0.4.6       | 25 KB     |            |   0% 
2022-11-02T14:07:48.3674967Z colorama-0.4.6       | 25 KB     | ########## | 100% 
2022-11-02T14:07:48.3675406Z 
2022-11-02T14:07:48.4560621Z ruamel_yaml-0.15.80  | 259 KB    |            |   0% 
2022-11-02T14:07:48.4561606Z ruamel_yaml-0.15.80  | 259 KB    | ########## | 100% 
2022-11-02T14:07:48.4562063Z 
2022-11-02T14:07:48.5170162Z certifi-2022.9.24    | 155 KB    |            |   0% 
2022-11-02T14:07:48.5171020Z certifi-2022.9.24    | 155 KB    | ########## | 100% 
2022-11-02T14:07:48.5171414Z 
2022-11-02T14:07:48.5518537Z python_abi-3.7       | 4 KB      |            |   0% 
2022-11-02T14:07:48.5519230Z python_abi-3.7       | 4 KB      | ########## | 100% 
2022-11-02T14:07:48.5519654Z 
2022-11-02T14:07:48.6037140Z charset-normalizer-2 | 36 KB     |            |   0% 
2022-11-02T14:07:48.6038040Z charset-normalizer-2 | 36 KB     | ########## | 100% 
2022-11-02T14:07:48.6040865Z 
2022-11-02T14:07:48.6698023Z urllib3-1.26.11      | 102 KB    |            |   0% 
2022-11-02T14:07:48.6699169Z urllib3-1.26.11      | 102 KB    | ########## | 100% 
2022-11-02T14:07:48.6701804Z 
2022-11-02T14:07:48.8990661Z conda-4.12.0         | 1.0 MB    |            |   0% 
2022-11-02T14:07:48.8991674Z conda-4.12.0         | 1.0 MB    | ########## | 100% 
2022-11-02T14:07:48.8994537Z conda-4.12.0         | 1.0 MB    | ########## | 100% 
2022-11-02T14:07:48.8994900Z 
2022-11-02T14:07:48.9600609Z pycparser-2.21       | 100 KB    |            |   0% 
2022-11-02T14:07:48.9601955Z pycparser-2.21       | 100 KB    | ########## | 100% 
2022-11-02T14:07:48.9604601Z 
2022-11-02T14:07:49.0108650Z yaml-0.2.5           | 87 KB     |            |   0% 
2022-11-02T14:07:49.0109374Z yaml-0.2.5           | 87 KB     | ########## | 100% 
2022-11-02T14:07:49.0109734Z 
2022-11-02T14:07:49.1023503Z libsolv-0.7.22       | 443 KB    |            |   0% 
2022-11-02T14:07:49.1024560Z libsolv-0.7.22       | 443 KB    | ########## | 100% 
2022-11-02T14:07:49.1027117Z 
2022-11-02T14:07:49.1693989Z cffi-1.15.1          | 227 KB    |            |   0% 
2022-11-02T14:07:49.1694792Z cffi-1.15.1          | 227 KB    | ########## | 100% 
2022-11-02T14:07:49.1695406Z 
2022-11-02T14:07:49.3242911Z conda-package-handli | 1.0 MB    |            |   0% 
2022-11-02T14:07:49.3243917Z conda-package-handli | 1.0 MB    | ########## | 100% 
2022-11-02T14:07:49.3246875Z conda-package-handli | 1.0 MB    | ########## | 100% 
2022-11-02T14:07:49.3247242Z 
2022-11-02T14:07:49.3741885Z tqdm-4.64.1          | 82 KB     |            |   0% 
2022-11-02T14:07:49.3742850Z tqdm-4.64.1          | 82 KB     | ########## | 100% 
2022-11-02T14:07:49.4208559Z Preparing transaction: ...working... done
2022-11-02T14:07:49.5132280Z Verifying transaction: ...working... done
2022-11-02T14:07:50.6261485Z Executing transaction: ...working... done
2022-11-02T14:07:56.7482421Z ##[group]Run cp environment.yml /tmp/environment.yml && sed -i -e 's/- python/- python=3.7/' /tmp/environment.yml
2022-11-02T14:07:56.7483045Z [36;1mcp environment.yml /tmp/environment.yml && sed -i -e 's/- python/- python=3.7/' /tmp/environment.yml[0m
2022-11-02T14:07:56.7536611Z shell: /usr/bin/bash -e {0}
2022-11-02T14:07:56.7536831Z env:
2022-11-02T14:07:56.7537018Z   PS1: (__setup_conda) 
2022-11-02T14:07:56.7537283Z   CONDA_PREFIX: /usr/share/miniconda/envs/__setup_conda
2022-11-02T14:07:56.7537558Z   CONDA_DEFAULT_ENV: __setup_conda
2022-11-02T14:07:56.7537816Z   CONDA_PROMPT_MODIFIER: (__setup_conda) 
2022-11-02T14:07:56.7538071Z   CONDA_EXE: /usr/share/miniconda/bin/conda
2022-11-02T14:07:56.7538301Z   _CE_M: 
2022-11-02T14:07:56.7538483Z   _CE_CONDA: 
2022-11-02T14:07:56.7538715Z   CONDA_PYTHON_EXE: /usr/share/miniconda/bin/python
2022-11-02T14:07:56.7538964Z ##[endgroup]
2022-11-02T14:07:56.7693373Z ##[group]Run mamba env update --name root --file /tmp/environment.yml
2022-11-02T14:07:56.7693756Z [36;1mmamba env update --name root --file /tmp/environment.yml[0m
2022-11-02T14:07:56.7744988Z shell: /usr/bin/bash -e {0}
2022-11-02T14:07:56.7745211Z env:
2022-11-02T14:07:56.7745418Z   PS1: (__setup_conda) 
2022-11-02T14:07:56.7745689Z   CONDA_PREFIX: /usr/share/miniconda/envs/__setup_conda
2022-11-02T14:07:56.7745952Z   CONDA_DEFAULT_ENV: __setup_conda
2022-11-02T14:07:56.7746213Z   CONDA_PROMPT_MODIFIER: (__setup_conda) 
2022-11-02T14:07:56.7746489Z   CONDA_EXE: /usr/share/miniconda/bin/conda
2022-11-02T14:07:56.7746708Z   _CE_M: 
2022-11-02T14:07:56.7746897Z   _CE_CONDA: 
2022-11-02T14:07:56.7747148Z   CONDA_PYTHON_EXE: /usr/share/miniconda/bin/python
2022-11-02T14:07:56.7747404Z ##[endgroup]
2022-11-02T14:07:57.0859873Z Traceback (most recent call last):
2022-11-02T14:07:57.0860722Z   File "/usr/share/miniconda/envs/__setup_conda/bin/mamba", line 7, in <module>
2022-11-02T14:07:57.0861428Z     from mamba.mamba import main
2022-11-02T14:07:57.0862960Z   File "/usr/share/miniconda/envs/__setup_conda/lib/python3.7/site-packages/mamba/mamba.py", line 53, in <module>
2022-11-02T14:07:57.0863636Z     from mamba.utils import get_index, to_package_record_from_subjson, _supplement_index_with_system
2022-11-02T14:07:57.0864569Z   File "/usr/share/miniconda/envs/__setup_conda/lib/python3.7/site-packages/mamba/utils.py", line 7, in <module>
2022-11-02T14:07:57.0865441Z     from .FastSubdirData import FastSubdirData
2022-11-02T14:07:57.0866179Z   File "/usr/share/miniconda/envs/__setup_conda/lib/python3.7/site-packages/mamba/FastSubdirData.py", line 23, in <module>
2022-11-02T14:07:57.0866558Z     from conda._vendor.auxlib.ish import dals
2022-11-02T14:07:57.0866960Z ModuleNotFoundError: No module named 'conda._vendor.auxlib'
2022-11-02T14:07:57.1364927Z ##[error]Process completed with exit code 1.
2022-11-02T14:07:57.1486469Z Post job cleanup.
2022-11-02T14:07:57.2737955Z [command]/usr/bin/git version
2022-11-02T14:07:57.2784307Z git version 2.38.1
2022-11-02T14:07:57.2821564Z Temporarily overriding HOME='/home/runner/work/_temp/7d668b4a-3421-4187-8c48-27b4c7978043' before making global git config changes
2022-11-02T14:07:57.2822324Z Adding repository directory to the temporary git global config as a safe directory
2022-11-02T14:07:57.2828224Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/snappy-pipeline/snappy-pipeline
2022-11-02T14:07:57.2869549Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-11-02T14:07:57.2903558Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-11-02T14:07:57.3141687Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-11-02T14:07:57.3159700Z http.https://github.com/.extraheader
2022-11-02T14:07:57.3178001Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2022-11-02T14:07:57.3203794Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-11-02T14:07:57.3660323Z Cleaning up orphan processes
```
Action example: https://github.com/bihealth/snappy-pipeline/actions/runs/3378175318

**Possible solution**
Pin `mamba` to last working version, v0.27.0.

**Additional context**
Related: https://github.com/mamba-org/mamba/issues/2078

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -26,7 +26,7 @@ jobs:
           lfs: true
           fetch-depth: 2
       - name: Install mamba.
-        run: conda install -y mamba
+        run: conda install -y mamba==0.27.0
       - name: Prepare environment.yml file.
         run: >
           cp environment.yml /tmp/environment.yml && sed -i -e

